.gitignore

```diff
@@ -29,3 +29,4 @@ yarn-debug.log*
 yarn-error.log*
 sdlschema
 sdlschema.graphql
+.eslintcache
```

Comment:
This file is annoying its part of eslint cant figure out a way to get it to stop generating.